apps/static/music/music-progression.json

```diff
@@ -43,10 +50,24 @@
           "message": "It's good that you played one sound, but you need to **play two more**."
         },
         {"message": "Please **play three sounds together**."}
-      ]
+      ],
+      "startSources": {
+        "blocks": {
+          "languageVersion": 0,
+          "blocks": [{"type": "when_run_simple2", "x": 32, "y": 32}]
+        },
+        "variables": [{"name": "currentTime"}, {"name": "i"}]
+      }
     },
     {
-      "text": "Try everything!"
+      "text": "Try everything!",
+      "startSources": {
+        "blocks": {
+          "languageVersion": 0,
+          "blocks": [{"type": "when_run_simple2", "x": 33, "y": 33}]
```

Comment:
Is there a reason we use slightly different x/y values in each version of startSources?

Reply:
Well-spotted!  This is a temporary measure which makes it easy for me to verify that the right sources have been loaded. :)